src/components/mc/cpu/reduce_alpha/mc_cpu_reduce_alpha_double.c

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2020-2021.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+#include "mc_cpu.h"
+#include "reduce_alpha/mc_cpu_reduce_alpha.h"
+
+ucc_status_t ucc_mc_cpu_reduce_multi_alpha_double(
+    const void *src1, const void *src2, void *dst, size_t n_vectors,
+    size_t count, size_t stride, ucc_reduction_op_t reduce_op,
+    ucc_reduction_op_t vector_op, double alpha)
+{
+    DO_DT_REDUCE_FLOAT(double, reduce_op, src1, src2, dst, n_vectors, count,
```

Comment:
@lappazos why do we need to re-defined DO_DT_REDUCE_FLOAT (and introduce all those code duplication) ? Why can't we just implement this ucc_mc_cpu_reduce_multi_alpha_double using DO_DT_REDUCE_FLOAT from mc_cpu_reduce.h + adding 1 more loop with multiplication by alpha? then we don't need that whole mc_cpu_reduce_alpha.h header